Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35658&attempt_id=5eaf84d8-f919-4449-a678-3aeec8e481c8)

<details><summary>Partial log (click to expand)</summary><p>

```
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:39 (message):
  Cannot determine Avro version number


-- Configuring incomplete, errors occurred!
See also "/build/avro-c-1.8.2/build/CMakeFiles/CMakeOutput.log".
builder for '/nix/store/ypzvdzfw7gjaxxvg13h47xn35izcz3mp-avro-c-1.8.2.drv' failed with exit code 1
[31;1merror:[0m build of '/nix/store/ypzvdzfw7gjaxxvg13h47xn35izcz3mp-avro-c-1.8.2.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/avro-c/default.nix
+++ b/pkgs/development/libraries/avro-c/default.nix
@@ -1,35 +1,30 @@
 { stdenv, bash, cmake, fetchurl, pkgconfig, jansson, zlib }:
 
-let version = "1.8.2"; in
-
-stdenv.mkDerivation rec {
+let
+  version = "1.8.2";
+in stdenv.mkDerivation rec {
   name = "avro-c-${version}";
 
-  #
   src = fetchurl {
     url = "mirror://apache/avro/avro-${version}/c/avro-c-${version}.tar.gz";
     sha256 = "03pixl345kkpn1jds03rpdcwjabi41rgdzi8f7y93gcg5cmrhfa6";
   };
 
-  patchPhase = ''
-    substituteInPlace version.sh \
-      --replace /bin/bash "$bash/bin/bash"
+  postPatch = ''
+    patchShebangs .
   '';
 
-  buildInputs = [
-    pkgconfig
-    cmake
-    jansson
-    zlib
-  ];
+  nativeBuildInputs = [ pkgconfig cmake ];
+
+  buildInputs = [ jansson zlib ];
 
   enableParallelBuilding = true;
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "A C library which implements parts of the Avro Specification";
     homepage = https://avro.apache.org/;
-    license = stdenv.lib.licenses.asl20;
-    maintainers = with stdenv.lib.maintainers; [ lblasc ];
-    platforms = stdenv.lib.platforms.all;
+    license = licenses.asl20;
+    maintainers = with maintainers; [ lblasc ];
+    platforms = platforms.all;
   };
 }

